src/util/manifest.js

```diff
@@ -46,6 +46,7 @@ export default async function getValidatedManifest(
   let manifestData;
 
   try {
+    manifestContents = stripBOM(manifestContents);
```

Comment:
I think it would be helpful to wrap this in its own try/catch so that the error is more specific:

```js
  try {
    manifestContents = stripBOM(manifestContents);
  } catch (error) {
    throw new InvalidManifest(
      `Error stripping BOM from manifest.json at ${manifestFile}: ${error}`);
  }
```